lib/class-names.js

```diff
@@ -0,0 +1,31 @@
+module.exports.default = {
+  'govspeak-address': 'govspeak-address h-adr',
```

Comment:
I think there’s a difference in how the Govspeak gem adds Microformats (i.e. `adr org fn` in a child `<div>`) and this package adds the equivalent Microformats 2 root `h-adr` class to the parent `<address>`. But this difference is probably beyond the scope of this PR.

Reply:
Yep. I spotted this too, but as far as I can see neither publishing components or the govspeak gem attach any styles to those classes (which is correct as they're not there for styling purposes) so that difference shouldn't matter